qp_shotgun/kneaddata/__init__.py

```diff
@@ -0,0 +1,86 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014--, The Qiita Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+
+from qiita_client import QiitaPlugin, QiitaCommand
+
+from .kneaddata import kneaddata
+
+__all__ = ['kneaddata']
+
+# Initialize the plugin
+plugin = QiitaPlugin(
```

Comment:
I'm not sure if this is going to work. Each repository should have a single "QiitaPlugin" instance. In case that we want to have multiple "QiitaPlugin" instances we need a configure and a start_* script for each of them, in which case I don't see the benefit of having all these tools in a single repository. I think this can be added as a command to the shotgun plugin.